Resources/Textures/Clothing/Mask/sadmime.rsi/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/dannno/-tg-station/blob/a51c5a85d327f562004cef2f36c081ad0d95fd9a/icons/obj/clothing/masks.dmi",
```

Comment:
link an actual TG link, not from a fork (same for the rest)

Reply:
corrected